plugins/wp-graphql-headless-webhooks/README.md

```diff
@@ -0,0 +1,27 @@
+# WPGraphQL Headless Webhooks
+
+## Overview
+
+A WordPress plugin that extends [WPGraphQL](https://www.wpgraphql.com/) to support webhook subscriptions and dispatching for headless WordPress environments. This allows external applications to listen and respond to WordPress content changes through GraphQL-driven webhook events.
+
+**⚠️ Work in Progress:** This plugin is under active development and not yet production-ready.
+
+## Features
+
+- Subscribe external services to WordPress content events
+- Dispatch webhook requests when specified GraphQL mutations or actions occur
+- Headless and API-friendly by design
+- Designed for modern decoupled WordPress applications
+
+## Requirements
+
+- WordPress 6.0+
+- WPGraphQL 1.8.0+
+- PHP 7.4+
+
+## Installation
+
+Clone the repository or download the latest release and place it in your WordPress `plugins` directory:
+
+## License
+BSD-0-Clause
```

Comment:
Definitely needs to be a different license. I'm assuming standard WP GPL 2?

Reply:
I will change